ci/ci.sh

```diff
@@ -292,6 +291,33 @@ test_python() {
   fi
 }
 
+# For running Python tests on Windows (excluding Serve).
+test_train_windows() {
```

Comment:
can we add a flag `is_flaky` in these functions and have a flaky list vs. non-flaky list of targets; then run the flaky ones with soft_fail?

Reply:
maybe in a follow up? I can even try use your `rayci` wrapper.

Reply:
these are just a copy of the `test_python` function, so that ml train tests can have their own target.

Reply:
sound good